src/FlipMove.js

```diff
@@ -69,7 +69,7 @@ class FlipMove extends Component {
     // can complete. Because we cannot mutate props, we make `state` the source
     // of truth.
     this.state = {
-      children: props.children.map(child => ({
+      children: Children.toArray(props.children).map(child => ({
```

Comment:
Yes, but in the `render` method of the prop converter, it renders a new `ComposedComponent` which is a `FlipMove` (as it should).  That means the converted props get passed back into the React virtual dom, which may modify the `children` prop before it gets to the `FlipMove` class.  Thus, a HOC cannot handle conversion of this particular prop; it must be done in `FlipMove`.